packages/palette-docs/src/components/MobileRenderer.tsx

```diff
@@ -0,0 +1,33 @@
+import * as Elements from "@artsy/palette"
+import { MDXProvider } from "@mdx-js/tag"
+import { ColorComponent } from "components/ColorComponent"
+import { MobilePreview } from "components/MobilePreview"
+import { CodeEditor, Playground } from "components/Playground"
+import MDXRenderer from "gatsby-mdx/mdx-renderer"
+import React from "react"
+import { Toggle as Toggler } from "react-powerplug"
+import { MarkdownComponents } from "./GlobalComponents"
+
+export const MobileRenderer = ({ children }) => {
+  return (
+    <MDXProvider components={MarkdownComponents}>
+      <MDXRenderer
+        scope={{
+          CodeEditor,
+          ColorComponent,
+          MobilePreview,
+          Playground,
+          Toggler,
+          ...Elements,
+          ...MarkdownComponents,
+          Box: require("@artsy/palette/dist/elements/Box/Box.ios").Box,
```

Comment:
The idea here is we override the web version of those components with the React Native version at runtime. I couldn't figure out a more elegant way to resolve those components since webpack is not aware of the `ios` platform extension. 

cc @damassi @alloy let me know if you can think of something. I was thinking a script that would generate this for all `ios.tsx` files in the src directory tree.

Reply:
cc @zephraph

Reply:
It'll be really hard to do at build time without building twice. While this approach works, my concern is that it would transparently break if we added a new `.ios` file for a component.